localstack/ext/java/mvnw

```diff
@@ -0,0 +1,286 @@
+#!/bin/sh
```

Comment:
Can we please remove the `mvnw` and `mvnw.cmd` files from the PR - I'd prefer not to have them in the repo. We can assume that anyone interested in running LocalStack via Maven will have `mvn` installed, hence no need for the wrapper script.